backend/app/services/implementations/tags_service.py

```diff
@@ -0,0 +1,25 @@
+from ..interfaces.tags_service import ITagsService
+from ...models.tag import Tag
+
+
+class TagsService(ITagsService):
+    """
+    Tags implementation with tags management methods
+    """
+
+    def __init__(self, logger):
+        """
+        Create an instance of TagsService
+
+        :param logger: application's logger instance
+        :type logger: logger
+        """
+        self.logger = logger
+
+    def get_tags(self):
+        try:
+            tags_results = Tag.query.all()
```

Comment:
I do not know if it is an issue but I noticed that it is getting all tags including ones with the status "Deleted".

![image](https://github.com/uwblueprint/supportive-housing/assets/43042601/67f0b8c4-02c1-4101-94d3-6110ff438e83)

Reply:
@owen-sellner this is fine for now, I've made a future ticket to remove this field